crates/red_knot_python_semantic/resources/mdtest/type_properties/is_subtype_of.md

```diff
@@ -1099,5 +1099,54 @@ static_assert(is_subtype_of(TypeOf[C.foo], object))
 static_assert(not is_subtype_of(object, TypeOf[C.foo]))
 ```
 
+### Classes with `__call__`
+
+```py
+from typing import Callable
+from knot_extensions import TypeOf, is_subtype_of, static_assert, is_assignable_to
+
+class A:
+    def __call__(self, a: int) -> int:
+        return a
+
+a = A()
+
+static_assert(is_subtype_of(A, Callable[[int], int]))
+static_assert(not is_subtype_of(A, Callable[[], int]))
+static_assert(not is_subtype_of(Callable[[int], int], A))
+
+def _(f: Callable[[int], int]) -> None: ...
+
+_(a)
```

Comment:
Nit: underscore is a good name for functions we are never calling or referencing, we just want a scope with some type-annotated names in it. It's not a good name for functions we actually reference.

```suggestion
def f(fn: Callable[[int], int]) -> None: ...

f(a)
```